### PR TITLE
[CPU functional/unit]Change cpu functional and unit test to snake case

### PR DESCRIPTION
**Pull request by xipingyan**
### Details:
 - *This is second PR*
 - *Binary names changed to **ov_cpu_unit_tests** and **ov_cpu_func_tests***

### Tickets:
 - *73613*

### CODE REVIEW

**Comment by xipingyan** [comment]
Hi All reviewers, 
Because this is name format changing, so you will see so many many files changes.
Please be patient, or let's CI to make sure my update correct.

**Comment by xipingyan** [comment]
I had renamed output test binary name.
cpuUnitTests -> cpu_unit_tests
cpuFuncTests -> cpu_func_tests

From the CI failed log, we could find CI script need to update, change cpu test name to  **cpu_unit_tests** and  **cpu_func_tests**.

hi @peterchen-intel and @wangleis , @riverlijunjie do you know who can provide help?

**Comment by peterchen-intel** [comment]
Change the binary name only

**Comment by xipingyan** [comment]
Now we just wait https://github.com/intel-innersource/frameworks.ai.openvino.ci.product-configs/pull/1340 ready.
If this PR pass, we need to merge https://github.com/openvinotoolkit/openvino/pull/12856 and https://github.com/intel-innersource/frameworks.ai.openvino.ci.product-configs/pull/1350 together.
Do not merge https://github.com/intel-innersource/frameworks.ai.openvino.ci.product-configs/pull/1340, it is only for verification.

File: .ci/azure/linux.yml [hunk 1]
```diff
@@ -354,7 +354,7 @@ jobs:
     displayName: 'IE UT'
     continueOnError: false
 
-  - script: $(RUN_PREFIX) $(INSTALL_TEST_DIR)/cpuUnitTests --gtest_output=xml:$(INSTALL_TEST_DIR)/TEST-cpuUnitTests.xml
+  - script: $(RUN_PREFIX) $(INSTALL_TEST_DIR)/ov_cpu_unit_tests --gtest_output=xml:$(INSTALL_TEST_DIR)/TEST-ov_cpu_unit_tests.xml
     displayName: 'CPU UT'
```
Comment (ilyachur) [inline .ci/azure/linux.yml:358]: I propose to rename targets too

Comment (ilyachur) [reply .ci/azure/linux.yml:358]: Intel CPU Unit Tests for example

Comment (xipingyan) [reply .ci/azure/linux.yml:358]: Sorry, a little confused, changing displayName from 'CPU UT' to 'Intel CPU Unit Tests'? @ilyachur

Comment (xipingyan) [reply .ci/azure/linux.yml:358]: cmake target name: TARGET_NAME is ov_cpu_unit_tests.
https://github.com/xipingyan/frameworks.ai.openvino.ci.product-configs/blob/master/conf_dldt_test.py#L83 has provided name format.

**Comment by xipingyan** [comment]
Fantastic, https://github.com/intel-innersource/frameworks.ai.openvino.ci.product-configs/pull/1340 all test passed, @ilyachur @wangleis Please check it again, if nothing to change, could you please help merge them?
1: https://github.com/intel-innersource/frameworks.ai.openvino.ci.product-configs/pull/1350
2: https://github.com/openvinotoolkit/openvino/pull/12856

**Do not merge** https://github.com/intel-innersource/frameworks.ai.openvino.ci.product-configs/pull/1340

**Comment by ilyachur** [comment]
@dmitry-gorokhov Please take a look

File: .ci/azure/linux.yml (hunk 1 above)
Comment (ilyachur) [reply .ci/azure/linux.yml:358]: Yes I am talking about more human readable `displayName`

**Review by dmitry-gorokhov — APPROVED** [review]
LGTM. Please resolve remaining comment

File: .ci/azure/linux.yml (hunk 1 above)
Comment (xipingyan) [reply .ci/azure/linux.yml:358]: Totally agree.
Changed cpu unit test name displayName from "CPU UT" to "Intel CPU Unit Tests".
For ov_cpu_func_tests's displayName: 'CPU FuncTests', I think it is readable, so don't change.

**Review by ilyachur — APPROVED** [review]

**Pull request merged**